src/test/java/org/breedinginsight/services/BrAPIGermplasmServiceUnitTest.java

```diff
@@ -141,10 +161,9 @@ public void getGermplasmListExport() {
 
         //Check file values
         assertEquals(listName+"_"+timestamp, downloadFile.getFileName(), "Incorrect export file name");
-        assertEquals(2, resultTable.rowCount(), "Wrong number of rows were exported");
         assertEquals(expectedColumnNames, resultTable.columnNames(), "Incorrect columns were exported");
-        assertEquals("Germplasm A", resultTable.get(0, 1), "Incorrect data exported");
-        assertEquals("2", resultTable.get(0, 6), "Incorrect data exported");
+        assertEquals(2, resultTable.rowCount(), "Wrong number of rows were exported");
+        // assertEquals("Germplasm A", resultTable.get(0, 1), "Incorrect data exported");
```

Comment:
Would be ideal if possible for these assertions to be uncommented so the test checks for correct data exported and not just number of rows.

Reply:
Done.  Tests are running now